### PR TITLE
Add superseded state to product cards in pricing page

### PR DESCRIPTION
**Pull request by monsieur-z**
### Changes proposed in this Pull Request

This PR marks the products that are made irrelevant by a user's subscription as such, in the pricing page. For instance, Backup Daily is of no interest when you own Complete, since Complete includes real-time backups.

Keeping the same example, Backup Daily is not technically included in Complete, but we'll show the existing "Included in plan" message to make it simpler and avoid confusion.

Fixes 1164141197617539-as-1201041449585202

### Implementation notes

I initially tried to check if a plan superseded the other by comparing features. However, this seemed brittle to me in the long run and decided to opt for a config map.

### Testing instructions

_Prerequisite: make sure you have a site with a Complete subscription, and another site with a different subscription_

- Download the PR and run Calypso
- Visit the pricing page with the Complete site
- Notice that Backup Daily, Security Daily, and Security Real-time are marked as included
- Verify that the rest of the page looks the same
- For good measure, select the other site, and check that products cards show the proper status, if any

### Screenshots
Screen Shot 2021-10-06 at 4 17 39 PM

### CODE REVIEW

File: client/components/jetpack/card/jetpack-product-card/display-price/superseded.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { Gridicon } from '@automattic/components';
+import { useTranslate } from 'i18n-calypso';
+
+const Superseded: React.FC = () => {
+	const translate = useTranslate();
+
+	return (
+		<p className="display-price__you-own-this">
+			<Gridicon className="display-price__you-own-this-icon" icon="checkmark-circle" size={ 48 } />
+			{ translate( 'Superseded by your plan' ) }
```
Comment (monsieur-z) [inline client/components/jetpack/card/jetpack-product-card/display-price/superseded.tsx]: I have some doubts about the copy here, not being a native speaker. Is that alright? Or would something else feel more natural?

Comment (mattgawarecki) [reply client/components/jetpack/card/jetpack-product-card/display-price/superseded.tsx]: This is grammatically correct, but I wonder if Design folks might have some better copy. "Superseded" is probably a bit too advanced of a word, but I couldn't think of anything better when I filed the issue 😅

**Review by eeeeevon13 — CHANGES_REQUESTED** [review]
The wording is supposed to be "Included in your plan". "Superceed" is a word that will be too complicated for some users. We want to keep our language very simple so it's understood at a glance.  Why is this being changed?

File: client/components/jetpack/card/jetpack-product-card/display-price/superseded.tsx (hunk 1 above)
Comment (eeeeevon13) [reply client/components/jetpack/card/jetpack-product-card/display-price/superseded.tsx]: Why are we changing this from "included in your plan"?

**Comment by mattgawarecki** [comment]
@eeeeevon13 In a technical sense there's a difference between "included" vs "superseded," but to the customer, I agree it's probably safe to stick with "included."

The reason for the technical difference is that in Calypso, we say a product is included in a plan if that product is explicitly a feature of the plan; e.g., Backup is included in Security. However, we haven't (up to now) accounted for a **plan** being included in another plan, like Security being a part of Complete.

**Comment by monsieur-z** [comment]
Thanks for the feedback. Let's make it the same for users then, that'll probably be less confusing indeed.

I'll add what we talked about for the tiers as well, @mattgawarecki.

**Comment by monsieur-z** [comment]
I've updated the code and the instructions to reflect the changes.

**Review by mattgawarecki — APPROVED** [review]
Everything tested well -- I used Complete and Security Real-time -- and all unit tests pass. The API for generating product cards continues to grow 😅, but these additions make sense, and the API itself is a wider issue I think we can address separately in the future. Approved, nice work! :shipit:

**Pull request merged**